source/users/custom-metadata.txt

```diff
@@ -1,8 +1,8 @@
-.. _custom-user-data:
+.. _custom-metadata:
```

Comment:
nice

Reply:
and good you included `custom-user-data` below. it can be dangerous to fully delete refs b/c they can be used by other repos (docs-realm for example).